### ISSUE TITLE
Cenarion Hold Reputation Formulas and Patterns not appearing in sales window

### ISSUE THREAD

**Issue by Bolrock1** [elysium-project/server#2371]
Hello and thank you for taking your time to read this report.
The Enchanting Supplies Vendor in Cenarion Hold does not appear to be selling any Formulas or Patterns which can be obtained by Reputation. I opened a ticket to which the response was to delete the WDB folder and restart the client. This did not resolve the issue
This screenshot was taken on Anathema the 02.09.17
![untitled1](https://user-images.githubusercontent.com/31574446/29998696-25ebc226-9032-11e7-8ec5-546daa231697.png)

**Comment by Kolmarwow** [comment]
This is intentional, none of the Cenarion Hold recipes (aside from a few leatherworking patterns) exist at the moment. They come in 1.11 with Naxxramas.

**Comment by Bolrock1** [comment]
I disagree, many people have the recipes already and say they bought them here.

**Comment by Komelansky** [comment]
As of today, 13th of October, this problem still persists for me. After deleting WDB and restarting client, I still had only the following recipes offered at Reputation honored:
![image](https://user-images.githubusercontent.com/32190915/31545495-edb4c61a-b01f-11e7-9015-94229d2d7c09.png)

**Comment by Komelansky** [comment]
![image](https://user-images.githubusercontent.com/32190915/31558870-e6d7e758-b04e-11e7-9390-f7f9635b6e08.png)

now second page as well, didnt upload before
